src/grafana/values/values.yaml

```diff
@@ -5,3 +5,30 @@ sidecar:
   datasources:
     enabled: true
     label: grafana_datasource
+
+extraSecretMounts:
+  - name: auth-generic-oauth-secret-mount
+    secretName: sso-client-uds-core-admin-grafana
+    defaultMode: 0440
+    mountPath: /etc/secrets/auth_generic_oauth
+    readOnly: true
+
+grafana.ini:
+  server:
+    root_url: https://grafana.admin.###ZARF_VAR_DOMAIN###
+  auth.generic_oauth:
+    # https://grafana.com/docs/grafana/latest/setup-grafana/configure-security/configure-authentication/keycloak/
+    enabled: true
+    client_id: $__file{/etc/secrets/auth_generic_oauth/clientId}
+    client_secret: $__file{/etc/secrets/auth_generic_oauth/secret}
+    scopes: openid profile
+    email_attribute_path: email
+    login_attribute_path: preferred_username
+    name_attribute_path: name
+    name: UDS Identity Service
+    auth_url: https://keycloak.admin.###ZARF_VAR_DOMAIN###/realms/uds/protocol/openid-connect/auth
+    token_url: http://keycloak-http.keycloak.svc.cluster.local:8080/realms/uds/protocol/openid-connect/token
```

Comment:
Is this just to avoid going through the gateway? Or just because internal DNS does not work?

Reply:
I was trying to keep all back-channel comms internal to the cluster and avoid going through the gw